packages/solid/package.json

```diff
@@ -0,0 +1,108 @@
+{
+  "name": "@uploadthing/solid",
+  "author": "OrJDev",
+  "license": "ISC",
+  "version": "1.0.0",
+  "type": "module",
+  "exports": {
```

Comment:
Shhiiiesh thats a long exports field

Reply:
This is auto generated by the tsup solid preset, it should work for the vite solid plugin and module & etc 😎